src/helper.js

```diff
@@ -75,7 +75,7 @@ function setHome(newPath) {
 	configPath = path.join(homePath, "config.js");
 	usersPath = path.join(homePath, "users");
 	storagePath = path.join(homePath, "storage");
-	packagesPath = path.join(homePath, "packages", "node_modules");
```

Comment:
`packagesPath` now does not include the `node_modules` part, because that means we need to compute the parent every time we want to access `package.json` of the packages folder. In practice, we should rarely have to rely on the `node_modules` location, `require` should do things nicely in _most_ case.